src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -22,7 +23,9 @@ public DeleteCommand parse(String args) throws ParseException {
             return new DeleteCommand(index);
         } catch (ParseException pe) {
             throw new ParseException(
-                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteCommand.MESSAGE_USAGE), pe);
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                            String.format(MESSAGE_CHECK_INPUT, pe.getMessage()) + "\n"
+                                    + DeleteCommand.MESSAGE_USAGE), pe);
```

Comment:
I remember we had this discussion right, `ledit 1 1 rate/40` throws invalid index error message because the parameter was spelled wrongly, not because the indices were invalid

Reply:
I think the behaviour should be as such:
- empty preamble: MESSAGE_INVALID_COMMAND_FORMAT + MESSAGE_USAGE
- preamble < 2 : MESSAGE_INSUFFICIENT_INDICES
- preamble > 2: iMESSAGE_INVALID_COMMAND_FORMAT + MESSAGE_USAGE
- if index !isNonZeroUnsignedInteger : `The index provided is invalid`

Reply:
For single index:
preamble > 1 : MESSAGE_INVALID_COMMAND_FORMAT + MESSAGE_USAGE